release-please-config.json

```
@@ -1,13 +1,19 @@
 {
   "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
   "draft-pull-request": true,
-  "include-component-in-tag": false,
+  "include-component-in-tag": true,
   "include-v-in-tag": true,
+  "separate-pull-requests": false,
   "packages": {
-    ".": {
-      "release-type": "rust",
-      "changelog-path": "CHANGELOG.md"
+    "post-compute": {
+      "component": "post-compute",
+      "package-name": "tee-worker-post-compute",
+      "release-type": "rust"
+    },
+    "pre-compute": {
+      "component": "pre-compute",
+      "package-name": "tee-worker-pre-compute",
+      "release-type": "rust"
     }
   }
```

Since all packages need a rust release we don't have to specify it individualy for each package
```suggestion
"release-type": "rust",
"packages": {
    "post-compute": {
      "component": "post-compute",
      "package-name": "tee-worker-post-compute"
    },
    "pre-compute": {
      "component": "pre-compute",
      "package-name": "tee-worker-pre-compute"
   }
}
```